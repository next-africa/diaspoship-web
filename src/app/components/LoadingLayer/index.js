//React
import React from 'react';
import PropTypes from 'prop-types';
//Material-UI
import { withStyles } from 'material-ui/styles';

const styles = theme => ({
  root: {
    position: 'absolute',
    top: 0,
    right: 0,
    bottom: 0,
    left: 0,
    zIndex: 999,
    backgroundColor: 'rgba(0,0,0,0.3)'
  }
});

const LoadingLayer = ({ classes, isLoading }) => (
  <div className={classes.root} />
);

LoadingLayer.propTypes = {
  classes: PropTypes.object
};

export default withStyles(styles)(LoadingLayer);
