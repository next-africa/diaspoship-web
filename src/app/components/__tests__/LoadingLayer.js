import React from 'react';
import LoadingLayer from '../LoadingLayer';
import renderer from 'react-test-renderer';

describe('LoadingLayer', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<LoadingLayer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
