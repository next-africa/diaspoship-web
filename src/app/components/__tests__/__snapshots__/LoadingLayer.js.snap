// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingLayer renders as expected 1`] = `
<div
  className="LoadingLayer-root-1"
/>
`;
